FABulous/fabric_generator/fabric_gen.py

```diff
@@ -2642,6 +2650,293 @@ def generateBitsStreamSpec(self) -> Dict[str, Dict]:
 
         return specData
 
+    def genIOBel(self, gen_ios: list[Gen_IO], bel_name: str) -> Bel | None:
```

Comment:
Maybe move this function to a new file, and let `self` be the writer. Or we can do the move later when we are doing the file splitting.

Reply:
I can move it to the fabric automation file, this might be a better fit.